internal/app/ui/cpwsarea/wsmap/pmap/psettings/screenshot.go

```diff
@@ -57,12 +63,30 @@ func (p *Panel) showScreenshot() {
 
 func (p *Panel) createScreenshot() {
 	p.sessionScreenshot.saving = true
-
-	width, height := p.editor.Dmm().MaxX*dmmap.WorldIconSize, p.editor.Dmm().MaxY*dmmap.WorldIconSize
+	selectedTool := tools.Selected()
+	grabCurrentlySelected := selectedTool.Name() == tools.TNGrab
+	bounds := util.Bounds{X1: 0, Y1: 0, X2: 0, Y2: 0}
```

Comment:
I mean that if u only need X and Y, you can do `boundX, boundY := 0, 0`

Reply:
the problem is that i need X2 and Y2 to calculate width and height

Reply:
so i dont think that is gonna work

Reply:
oh wait i misread that

Reply:
done